contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -687,7 +687,9 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 
        FROM sys.pg_namespace_ext nc LEFT JOIN sys.babelfish_namespace_ext ext ON nc.nspname = ext.nspname,
             pg_proc p inner join sys.schemas sch on sch.schema_id = p.pronamespace
-	    inner join sys.all_objects ao on ao.object_id = CAST(p.oid AS INT),
+	    inner join sys.all_objects ao on ao.object_id = CAST(p.oid AS INT)
+		left join sys.babelfish_function_ext f on p.proname = f.funcname and sch.schema_id::regnamespace::name = f.nspname
+			and sys.babelfish_get_pltsql_function_signature(p.oid) = f.funcsignature collate sys.database_default,
```

Comment:
Can we just use `regnamespace` and function signature for the join and remove name since schema + signature is sufficient and make primary.

Reply:
As discussed offline, I will file a new JIRA to track this separately.